<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomecheck</class>
 <widget class="QDialog" name="welcomecheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>库存查询</string>
    </attribute>
    <widget class="QLineEdit" name="leSearchname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearchGoods">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>21</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>货物名称：</string>
     </property>
    </widget>
    <widget class="QTableView" name="tvGoods">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>491</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCheckExit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>进货单查询</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>货单处理情况：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearchList">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QTableView" name="tvlist">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>481</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbModify">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCheckExit_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbDeal">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>进出货单：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbListType">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>进货单</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>出货单</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;审核员&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCheckExit</sender>
   <signal>clicked()</signal>
   <receiver>welcomecheck</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCheckExit_2</sender>
   <signal>clicked()</signal>
   <receiver>welcomecheck</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbModify</sender>
   <signal>clicked()</signal>
   <receiver>welcomecheck</receiver>
   <slot>pbModify_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSearchGoods</sender>
   <signal>clicked()</signal>
   <receiver>welcomecheck</receiver>
   <slot>pbSearchGoods_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSearchList</sender>
   <signal>clicked()</signal>
   <receiver>welcomecheck</receiver>
   <slot>pbSearchList_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pbSearchList_clicked()</slot>
  <slot>pbSearchGoods_clicked()</slot>
  <slot>pbModify_clicked()</slot>
 </slots>
</ui>
